content/applications/sales/sales/invoicing/milestone.rst

```diff
@@ -2,32 +2,244 @@
-    :align: center
+The :guilabel:`Project Template` field provides template options to use for the project that will be
+created when this specific product is ordered.
+
+The :guilabel:`Workspace Template` field (for the *Documents* app, not the *Projects* app) provides
+template options to use for the workspace that will be automatically generated for the project when
+this specific product is ordered.
+
+.. tip::
+   For organizational purposes, click the :guilabel:`Sales` tab on the product form, and enter a
+   custom 'Milestone' related descriptor in the :guilabel:`Sales Description` field. This
+   information appears in the :guilabel:`Description` column on the :guilabel:`Order Lines` tab of
+   the sales order.
+
+   Or, directly edit/modify the :guilabel:`Description` field on the :guilabel:`Order Lines` tab of
+   the sales order.
+
+   This is *not* a requirement.
 
 Invoice milestones
 ==================
 
-From the sales order, you can manually edit the quantity delivered as
-you complete a milestone.
+.. note::
+   The following flow features a trio of milestone products that have :guilabel:`Service` set as
+   their :guilabel:`Product Type`, and :guilabel:`Task` set on their :guilabel:`Create on Order`
+   field.
+
+    .. image:: milestone/settings-for-workflow.png
+       :align: center
+       :alt: Product with Service "Product Type" and "Task" in the Create on Order field on form.
+
+    Those tasks are then attached to a pre-existing :guilabel:`Project`, which, in this case, is
+    titled, :guilabel:`Rebranding Projects`.
+
+To invoice milestones, create a sales order with the milestone product(s). To do that, go to
+:menuselection:`Sales app --> New`. Doing so reveals a blank quotation form.
+
+From this quotation form, add a :guilabel:`Customer`. Then, click :guilabel:`Add a product` in the
+:guilabel:`Order Lines` tab. Next, add the milestone product(s) to the :guilabel:`Order Lines` tab.
+
+Once the corresponding milestone product(s) have been added, click :guilabel:`Confirm` to confirm
+the order, which turns the quotation into a sales order.
+
+When the order is confirmed, new smart buttons appear at the top of the sales order based on what
+was selected in the :guilabel:`Create on Order` field on the product form.
+
+.. image:: milestone/confirmed-sales-order-milestone.png
+   :align: center
+   :alt: Confirmed sales order with milestone products in the order lines tab.
+
+From the sales order, click the :guilabel:`Milestones` smart button. Doing so reveals a populated
+:guilabel:`Milestones` page. Click :guilabel:`New` to add milestones.
+
+.. image:: milestone/adding-milestones.png
+   :align: center
+   :alt: Adding milestones to a sales order with milestone products.
+
+Enter a :guilabel:`Name` for the milestone. Next, apply it to the corresponding :guilabel:`Sales
+Order Item`. Lastly, assign a :guilabel:`Deadline` to the milestone, if desired.
+
+.. note::
+   The :guilabel:`Quantity` field is not editable.
```

Comment:
Again, like my previous comment, but I might have missed a step so my flow is all wonky now, but I manually created a new milestone using the New button, and i was able to edit the Quantity field (16.3 runbot)
![image](https://github.com/odoo/documentation/assets/23611711/f34b6a4f-918f-4160-81aa-85f9b363af14)

Edited it again:
![image](https://github.com/odoo/documentation/assets/23611711/3034fd43-8995-4717-9e6a-ecf24f0b5253)

Reply:
Thanks for pointing this out! For the sake of avoiding any confusion, I'll go ahead and remove this minor note admonition. I really appreciate the in-depth feedback, @Felicious